Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -123,4 +125,23 @@ public function storeCodeTrekApplicantFeedback($data)
         $feedback->posted_on = date('y-m-d');
         $feedback->save();
     }
+
+    public function getCandidateFeedbacks($data)
+    {
+        $feedbacks = CodeTrekCandidateFeedback::where('candidate_id', $data['applicant'])->get();
+        $candidateFeedbacks = [];
+        foreach ($feedbacks as $feedback) {
+            $candidateFeedback = new \stdClass();
+            $candidateFeedback->id = $feedback['id'];
+            $candidateFeedback->posted_by = User::select('name')->where('id', $feedback['posted_by'])->first()->name;
+            $candidateFeedback->posted_on = $feedback['posted_on'];
+            $candidateFeedback->feedback_category = CodeTrekFeedbackCategories::select('category_name')->where('id', $feedback['category_id'])->first()->category_name;
```

Comment:
I think, we don't need to add this `first()`.
can you write like this? And please test this locally once, and let me know, it's works or not.
```suggestion
            $candidateFeedback->feedback_category = CodeTrekFeedbackCategories::select('category_name')->where('id', $feedback['category_id'])->category_name;
```

Reply:
I tried it before and it does not work, as the part before 'first()' leaves a eloquent builder which does not has a property 'name'.